Sample Project/TestBed/MSColorPicker/MSColorComponentView.h

```diff
@@ -0,0 +1,69 @@
+//
+// MSColorComponentView.h
+//
+// Created by Maksym Shcheglov on 2014-02-12.
+//
+// The MIT License (MIT)
+// Copyright (c) 2015 Maksym Shcheglov
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import <UIKit/UIKit.h>
+
+@class MSSliderView;
```

Comment:
Again, I am concerned about including third-party components here when not absolutely necessary. For example, this color choice problem can be easily solved with a few lines of our own code that provide a few standard values... rather than relying on external code which may cause issues, need to be rewritten, updated, etc. at some point.